components/contribution-flow/ContributionBlocker.js

```diff
@@ -49,6 +51,16 @@ const msg = defineMessages({
 /**
  * From received params, see if there's anything preventing the contribution
  */
+export const Redirect = ({ to }) => {
```

Comment:
This component is not really related to `ContributionBlocker`, so I think it deserves its own file (ie. `.components/Redirect.js`)